ignite/pkg/cosmosgen/templates/root/package.json.tpl

```diff
@@ -10,23 +10,25 @@
       "url": "http://www.apache.org/licenses/LICENSE-2.0"
     }
   ],
-  "main": "index.ts",
+  "main": "lib/index.js",
   "publishConfig": {
     "access": "public"
   },
-  "dependencies": {
-    "@cosmjs/launchpad": "0.27.0",
-    "@cosmjs/proto-signing": "0.27.0",
-    "@cosmjs/stargate": "0.27.0",
+  "scripts": {
+    "build": "NODE_OPTIONS='--max-old-space-size=16384' tsc",
```

Comment:
I'm not sure here, but is it really needed to send the NODE_OPTIONS?
If by chance there are other options defined for NODE_OPTIONS in the user environment this would override those. Also I think it might be an issue in machines with 8GB.

Reply:
for some reason I am unaware of, building the client fails with the defaults.